nixos/modules/services/monitoring/netdata.nix

```diff
@@ -0,0 +1,77 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.netdata;
+
+  configFile = pkgs.writeText "netdata.conf" cfg.configText;
+
+  defaultUser = "netdata";
+
+in {
+  options = {
+    services.netdata = {
+      enable = mkEnableOption "netdata";
```

Comment:
I believe `mkEnableOption` has fallen out of favour so probably shouldn't be used for new code.